community/shell/src/test/java/org/neo4j/shell/StartClientTest.java

```diff
@@ -212,6 +217,32 @@ public void shouldPrintVersionAndExit() throws Exception
         assertThat( version, startsWith( "Neo4j Community, version " ) );
     }
 
+    @Test
+    public void shouldNotStartBolt() throws IOException
+    {
+        // Given
+        AssertableLogProvider log = new AssertableLogProvider();
+
+        // When
+        new StartClient( System.out, System.err )
+        {
+            @Override
+            protected GraphDatabaseShellServer getGraphDatabaseShellServer( File path, boolean readOnly, String
+                    configFile ) throws RemoteException
+            {
+                return new GraphDatabaseShellServer(
+                        new GraphDatabaseFactory().setUserLogProvider( log ), path, readOnly, configFile );
+            }
+        }.start( new String[]{
+                        "-c", "RETURN 1;",
+                        "-path", db.getGraphDatabaseAPI().getStoreDir(),
+                        "-config", getClass().getResource( "/config-with-bolt-connector.conf" ).getFile()},
+                mock( CtrlCHandler.class ) );
+
+        // Then
+        log.assertNone( inLog( startsWith( Sessions.class.getPackage().getName() ) ).any() );
```

Comment:
Assert that we have not gotten any log lines from any class in the bolt packages.